src/java.xml/share/conf/jaxp-compat.properties

```diff
@@ -0,0 +1,116 @@
+################################################################################
+#           JAXP Compatibility Configuration File
+#
+# ** Warning: unsafe configuration, use with security assessment **
+#
+# jaxp-compat.properties is a JAXP configuration file with compatible settings
+# prior to a strict configuration such as jaxp-strict.properties. In this
+# configuration, properties that have more restrictive settings as in the
+# strict configuration (jaxp-strict.properties) are reversed back to their
+# defaults. In particular:
+#   - JDKCatalog Resolve is set to "continue"
+#   - Extension Functions are enabled
+#   - JAXP Limits are set to their default values
+#
+# This configuration file can be used to reverse back to a working environment
+# prior to any more restrictive configuration that may have been applied.
```

Comment:
How does this differ from jaxp.properties for JDK 23?  I can understand for when we move to secure by default, we just need to be clear on the purpose of each jaxp properties files